starter-code/lib/Password.js

```diff
@@ -0,0 +1,63 @@
+
+
+
+//Rules we will enforce on our users' passwords:
+
+//No character can be used more than once.
+////Only digits are allowed. No letters or special characters or anything else.
+//We will cut all passwords down to 10 characters if the user gives us a password that exceeds that length.
+//Unique characters: Create a function that checks that each character only appears once in the password. It should receives a password and display Ouch, bad password. in the console if there are any repeated characters. Otherwise, it should display Good password. in the console.
+
+var goodPsswd = "1234567890";
+var badPsswd = "1123456";
+var noRepeatChar = function (password, item) {
+for (i=0; i<password.length; i++) {
```

Comment:
Intenta no dejar espacio en el bucle for () -> for().
Declara la variable i dentro del for.  for(var i = 0...)